src/tools/pontoon/ui.md

```diff
@@ -4,115 +4,125 @@
 
 ## Pontoon translation workspace
 
-Pontoon’s translation workspace features the list of strings in the left sidebar and the editing space on the right.
+Pontoon’s translation workspace is organized in 3 columns:
```

Comment:
We mention three columns, than talk about main toolbar and then continue explaining more details of the three columns. Maybe mention the main toolbar here, too?

"Pontoon’s translation workspace consists of the main toolbar and 3 columns". Not ideal, but IMO more coherent.